### ISSUE TITLE
Engine for plugins or Apps?

### ISSUE THREAD

**Issue by juan-manuel-alberro** [Postleaf/postleaf#55]
Hello again, 

There's any plan in the roadmap to create a plugin (or app) system so we can extend the functionality of the CMS?

Maybe you can use the [project](https://github.com/Postleaf/postleaf/projects) tab to define next ideas 😄 

Cheers.

**Comment by claviska** [comment]
Yes, after 1.0 stable. I'm using milestones at the moment but will likely use projects for the next phase of dev.
